systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/AdminClientOperations.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources.crd.kafkaclients;
+
+public enum AdminClientOperations {
+    CREATE_TOPICS("create"),
+    DELETE_TOPICS("delete"),
+    LIST_TOPICS("list"),
+    UPDATE_TOPICS("update"),
```

Comment:
Would it make sense to rename also the operations in the admin client itself?

Reply:
Currently - I don't think it is needed, as there are no other operations needed afaik. 
If there will be a need for it, I would agree with your suggestion. 
Do you foresee any future usage? So far nobody needed KafkaAdminClient in 2+ years, so I am a bit sceptical.

Reply:
IDK if there be some other operations etc. in the future -> but when you renamed the enums, it would be great to rename also the operations. Let's keep it like this and change it in future, if it will be needed.